control/servos_mix_birotor.h

```diff
@@ -0,0 +1,122 @@
+ * may be used to endorse or promote products derived from this software without
+ * specific prior written permission.
+ * 
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" 
+ * AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE 
+ * IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE 
+ * ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE 
+ * LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR 
+ * CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF 
+ * SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS 
+ * INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN 
+ * CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) 
+ * ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE 
+ * POSSIBILITY OF SUCH DAMAGE.
+ ******************************************************************************/
+
+/*******************************************************************************
+ * \file servos_mix_quadcopter_diag.h
+ * 
+ * \author MAV'RIC Team
+ * \author Julien Lecoeur
+ *   
+ * \brief Links between torque commands and servos PWM command for quadcopters 
+ * in diagonal configuration
+ *
+ ******************************************************************************/
+
+
+
+
+
+#ifdef __cplusplus
+	extern "C" {
+#endif
+
+
+#include "control_command.h"
+#include "servos.h"
+#include "constants.h"
+#include "daler_dc_motor_ctrl.h"
+
+typedef struct
+{
+	uint8_t 	motor_left;
+	uint8_t 	motor_right;
+	uint8_t 	servo_left;
+	uint8_t		servo_right;
+	rot_dir_t 	motor_left_dir;
+	rot_dir_t 	motor_right_dir;
+	rot_dir_t 	servo_left_dir;
+	rot_dir_t 	servo_right_dir;
+	float 		min_thrust;
+	float		max_thrust;
+	float 		min_servo;
+	float		max_servo;
+	
+} servo_mix_birotor_conf_t;
+
+
+/**
+ * \brief	servos mix structure
+ */
+typedef struct 
+{	
+	uint8_t 	motor_left;
+	uint8_t 	motor_right;
+	uint8_t 	servo_left;
+	uint8_t		servo_right;
+	rot_dir_t 	motor_left_dir;
+	rot_dir_t 	motor_right_dir;
+	rot_dir_t 	servo_left_dir;
+	rot_dir_t 	servo_right_dir;
+	float 		min_thrust;
+	float		max_thrust;
+	float 		min_servo;
+	float		max_servo;
+	const torque_command_t* torque_command;
+	const thrust_command_t* thrust_command;
+	servos_t*         		servos;
+	daler_dc_motor_ctrl_t* 	dc_motors;
+} servo_mix_birotor_t;
+
+
+/**
+ * @brief [brief description]
+ * @details [long description]
+ * 
+ * @param servo_mix [description]
+ * @param config [description]
+ * @param torque_command [description]
+ * @param servo_pwm [description]
+ */
+void servo_mix_birotor_init(servo_mix_birotor_t* mix, const servo_mix_birotor_conf_t* config, const torque_command_t* torque_command, const thrust_command_t* thrust_command, servos_t* servos, daler_dc_motor_ctrl_t* 	dc_motor);
+
+
+/**
+ * @brief [brief description]

```

Comment:
Fixed in https://github.com/lis-epfl/MAVRIC_Library/commit/d7b866d6844bd2853c09ec29dcc40f3689ac24c9